sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1543,6 +1543,8 @@ pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
     return ReturnValue(Device->Platform);
   case PI_DEVICE_INFO_VENDOR_ID:
     return ReturnValue(pi_uint32{Device->ZeDeviceProperties.vendorId});
+  case PI_DEVICE_INFO_UUID:
+    return ReturnValue(Device->ZeDeviceProperties.uuid.id);
```

Comment:
This will return "uint8_t id[ZE_MAX_DEVICE_UUID_SIZE]" not the std::array. So let's "uin8_t *" be the PI interface.

Reply:
ok, will modify.